src/themes/default/elements/icon.overrides

```diff
@@ -201,6 +202,19 @@ i.icon.other.gender.horizontal:before { content: "\f22b"; }
 i.icon.neuter:before { content: "\f22c"; }
 i.icon.genderless:before { content: "\f22d"; }
 
+/* Accessibility */
+i.icon.universal.access:before { content: "\f29a"; }
+i.icon.alternative.wheelchair:before { content: "\f29b"; }
+i.icon.blind:before { content: "\f29d"; }
+i.icon.audio.description:before { content: "\f29e"; }
+i.icon.volume.control.phone:before { content: "\f2a0"; }
+i.icon.braille:before { content: "\f2a1"; }
```

Comment:
New icon: `braille`, by far the most known assistive technology. Depicts the letters "F A".